generators/lib/distros/openatrium.js

```diff
@@ -20,9 +21,17 @@ function init() {
   module.description = 'This project is built on [' + module.option.name + '](http://openatrium.com) for more information visit the [Atrium Project Homepage](https://drupal.org/project/openatrium).';
 
   module.releaseVersion = function(majorVersion, done, cb) {
-    require('../../lib/drupalProjectVersion').latestReleaseStable(module.id, majorVersion, done, cb);
+    require('../drupalProjectVersion').latestReleaseStable(module.id, majorVersion, done, cb);
```

Comment:
As before, I'd like to see this handled with a variable like we do with the "gadget" and "util" handling above.  Or, I also wonder why this function isn't just added to "util".